tabbycat/tournaments/templates/round_advance_check.html

```diff
@@ -45,31 +44,32 @@
   {% if increment_ok %}
     <div class="alert alert-success">
       <i data-feather="alert-circle"></i>
-      It looks like all the ballots from {{ round.name }} are confirmed, you should be good to go.
+      {% blocktrans trimmed %}It looks like all the ballots from {{ round.name }} are confirmed, you should be good to go.{% endblocktrans %}
```

Comment:
Tags with dots in them aren't allowed inside `blocktrans` tags, you need to use with `with` syntax to defined variables: `{% blocktrans trimmed with round_name=round.name %}`